Azure/azure-linux/ingest/parser.yml

```diff
@@ -0,0 +1,282 @@
+name: untitled-name
+pipeline:
+  - name: json_event
+    description: ''
+    filter: ''
```

Comment:
```suggestion
```

Are these fields mandatory?

Reply:
The parser was creating using the custom intake UI, which create some fields even if they are empty. I will remove them